core/trie/trie.go

```diff
@@ -373,13 +425,13 @@ func (t *Trie) Put(key, value *felt.Felt) (*felt.Felt, error) {
 }
 
 // Put updates the corresponding `value` for a `key`
-func (t *Trie) PutWithProof(key, value *felt.Felt, lProofPath, rProofPath []StorageNode) (*felt.Felt, error) {
+func (t *Trie) PutWithProof(key, value *felt.Felt, proof []*StorageNode) (*felt.Felt, error) {
```

Comment:
`proof` contains both the left and right proof, right?

Reply:
Yup